libevm/EVMC.cpp

```diff
@@ -21,6 +21,36 @@ EVM::EVM(evmc_instance* _instance) noexcept : m_instance(_instance)
         m_instance->set_option(m_instance, pair.first.c_str(), pair.second.c_str());
 }
 
+EVMC::EVMC(evmc_instance* _instance) : EVM(_instance)
+{
+    static constexpr auto tracer = [](evmc_tracer_context * context, int depth, int step,
```

Comment:
parameter names should follow cpp naming convention, `_statusCode` etc.

Also, consider making it a free function or static method, it's pretty complicated for a lambda. And it would be easier to find tracing function in the code.